tests/run_tests.sh

```diff
@@ -0,0 +1,10 @@
+INCLUDED_NBS=$(find ./nbdev_nbs -name "*.ipynb")
+python -m pytest --nbmake $(echo $INCLUDED_NBS)
+
+
+INCLUDED_NBS=$(find ./docs/nbs -name "*.ipynb" | grep -v tutorial_dev_spectral_libraries.ipynb)
```

Comment:
did you drop the `TODO` comments on purpose? I thing they served well explaining why we exclude some notebooks here ..

Reply:
Yes, I will move them back